web/js/date/wheels.js

```diff
@@ -89,3 +89,5 @@ export function dateWheels(models, config) {
   init();
   return self;
 };
+
+export default dateWheels;
```

Comment:
@localjo what's the reason for making this a default export instead of exporting the `function`?

Reply:
`dateWheels` is a function. These two are more or less equivalent;

```
export default function(){};
```

And

```
var dateWheels = function(){};
export default dateWheels;
```

I don't remember exactly why I did it this way here. Probably just to be clearer by giving a name to what I was exporting.